.github/workflows/acquisition-events-api.yml

```diff
@@ -62,4 +62,6 @@ jobs:
 
       - name: Build and upload to RiffRaff
         run: |
+          export LAST_TEAMCITY_BUILD=20000
+          export GITHUB_RUN_NUMBER=$(( $GITHUB_RUN_NUMBER + $LAST_TEAMCITY_BUILD ))
```

Comment:
I think, without these lines, the acquisition-events-api was getting the build number generated by the  support-lambdas-build.yml

Reply:
How did you get the 20,000 value, out of curiosity? Picked a value bigger than the most recent TeamCity build?

Reply:
> How did you get the 20,000 value, out of curiosity? Picked a value bigger than the most recent TeamCity build?

yes